fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/PrivateBrowsingDescriptionViewHolder.kt

```diff
@@ -121,3 +136,56 @@ private fun PrivateBrowsingDescriptionPreview() {
         )
     }
 }
+
+/**
+ * Total Private Browsing Mode description.
+ *
+ * @param onLearnMoreClick Invoked when the user clicks on the who can see my activity link.
+ */
+@Composable
+fun FeltPrivacyModeDescription(
+    onLearnMoreClick: () -> Unit,
+) {
+    Card(
+        modifier = Modifier
+            .wrapContentHeight(),
+        shape = RoundedCornerShape(8.dp),
+        backgroundColor = FirefoxTheme.colors.layerAccentNonOpaque,
+        elevation = 0.dp,
+    ) {
+        Column(
+            modifier = Modifier.padding(PaddingValues(16.dp)),
+        ) {
+            Text(
+                text = stringResource(id = R.string.felt_privacy_desc_card_title),
+                style = FirefoxTheme.typography.headline7,
+                color = FirefoxTheme.colors.textPrimary,
+                lineHeight = 20.sp,
```

Comment:
Do you have a figma link for the designs anywhere? I couldn't find it in any of other bugs nor meta. I think it comes in handy to have the figma link somewhere to help verify some of the styles. It also serves as a nice reference if we ever need to come back to the work here a year from now to see what the old designs were as well.

I am drawing my attention to the `lineHeight` here because we usually have the line height already defined in the text style `headline7`. So, it feels strange to me that we are deviating from the design system so I think it's worth checking in on if this is intended from UX.

Reply:
I manage to find the figma in the Jira epic. I don't believe the line height is needed based on what I am seeing:

<img width="1718" alt="Screen Shot 2023-08-16 at 3 31 14 PM" src="https://github.com/mozilla-mobile/firefox-android/assets/1190888/cf8999bc-4a4e-4158-85f4-f4c26c226a8d">

https://www.figma.com/file/Ha2T4I0nX0h7B95GYxH6wV/v1-ABM-minimal-UI?type=design&node-id=2585-15060&mode=design&t=0BZdkCV8g6spHjJA-0

Reply:
As a small request, could you add the figma link in the issues and meta?